users/views.py

```diff
@@ -1,3 +1,59 @@
-from django.shortcuts import render
+import json, re, bcrypt, jwt, requests
 
-# Create your views here.
+from django.shortcuts import render, redirect
+from django.views     import View
+from django.http      import JsonResponse
+from django.conf      import settings
+
+from users.models     import User
+
+class KakaoAPI:
+    def __init__(self, authorization_code):
+        self.authorization_code = authorization_code
+        self.token_url = "https://kauth.kakao.com/oauth/token"
+        self.user_url  = "https://kapi.kakao.com/v2/user/me"
+        self.redirect_url = "http://10.58.7.130:8000/users/kakao/callback"
```

Comment:
`redirect_url`은 어디서 사용되나요?

```suggestion
        self.redirect_url = "http://10.58.7.130:8000/users/kakao/callback"
```

Reply:
토큰을 요청하는 함수에서 사용됩니다ㅏ. 추후에 다른 곳에서도 사용하게 될 경우 redirect_url 수정의 편의성도 고려했습니다.